core/frontend/src/components/mavlink-inspector/MavlinkInspector.vue

```diff
@@ -158,9 +163,17 @@ export default Vue.extend({
       messages_in_view: [] as any[],
       selected_message_types: [],
       detailed_message: null as (null | any),
+      message_filter: '',
     }
   },
   computed: {
+    filtered_messages(): string[] {
+      try {
+        return this.message_types.filter((name: string) => name.match(this.message_filter.trim()))
```

Comment:
```suggestion
        return this.message_types.filter((name: string) => name.test(this.message_filter.trim()))
```

Maybe test() makes more sense here?

Reply:
`test` does not exist for `string`, only for `RegExp`. `match` was avoided to create a more simpler approach over creating a regex component.